experiment/reporter.py

```diff
@@ -37,18 +38,29 @@ def get_reports_dir():
     return exp_path.path('reports')
 
 
+def get_core_fuzzers():
+    """Return list of core fuzzers to be used for merging experiment data."""
+    return yaml_utils.read(CORE_FUZZERS_YAML)['fuzzers']
+
+
 def output_report(experiment_config: dict,
                   in_progress=False,
                   coverage_report=False):
     """Generate the HTML report and write it to |web_bucket|."""
     experiment_name = experiment_utils.get_experiment_name()
-    web_filestore_path = posixpath.join(experiment_config['report_filestore'],
-                                        experiment_name)
-
     reports_dir = get_reports_dir()
 
-    core_fuzzers = yaml_utils.read(CORE_FUZZERS_YAML)['fuzzers']
-    fuzzers = sorted(set(experiment_config['fuzzers']).union(set(core_fuzzers)))
+    core_fuzzers = set(get_core_fuzzers())
+    experiment_fuzzers = set(experiment_config['fuzzers'])
+    fuzzers = sorted(experiment_fuzzers.union(core_fuzzers))
+
+    # Calculate path to store report files in filestore.
+    web_filestore_path = experiment_config['report_filestore']
+    if experiment_fuzzers != core_fuzzers:
```

Comment:
this puts reports only for "full experiments" in the main directory, right? should we put reports there too where the experiment_fuzzers is a subset of the core fuzzers, but the report would merge the latest results for the rest of the core fuzzers? (e.g., an experiment that only runs afl and libfuzzer)

Reply:
Good paint, forgot about merging, so yes they can go in official, fixed in patch.